xapian-core/cluster/euclidian_sim.cc

```diff
@@ -40,86 +40,10 @@ EuclidianDistance::get_description() {
     return "Euclidian Distance metric";
 }
 
+/** TODO
+ *  Implement the euclidian distance metric for the current API
+ */
```

Comment:
I agree about having multiple distance measures, but what I mean is that if it isn't working, the entire class shouldn't be present. Each commit should represent a good state for the code, wherever possible. This isn't, because it contains options that people can compile against but don't make any sense — removing the `EuclidianDistance` class (for the time being) should solve this.

The second part of my comment is still relevant.